ml_model/src/main/java/edu/wm/sealab/featureextraction/Parser.java

```diff
@@ -47,8 +55,26 @@ public static void main(String[] args) {
       pw.append("parenthesis");
       pw.append(",");
       pw.append("literals");
+      pw.append(",");
+      pw.append("avgComments");
+      pw.append(",");
+      pw.append("avgComparisons");
+      pw.append(",");
+      pw.append("avgOperators");
+      pw.append(",");
+      pw.append("avgConditionals");
       pw.append("\n");
 
+      List<String[]> lines = null;
+      // try ( FileReader fileReader = new FileReader("ml_model/raw_loc_data.csv"); // uncomment for raw features
+      //       CSVReader csvReader = new CSVReaderBuilder(fileReader).withSkipLines(1).build();) {
```

Comment:
As a general rule, it's better to avoid commenting out code. If it's not supposed to be here anymore, delete it. If you really want to save it (maybe you might need to go back to it later), use the VCS: that's why we have git.

Reply:
The file has other existing lines similar to this that were there before. Should I remove those as well?
Should we make a Raw_Parser.java, or have static variables that tell Parser.java what paths to use based on whether we are analyzing the raw data or not? @NGimhana

Reply:
The ideal way should be using Java args and the same Parser for both the raw and non-raw features. This is a separate task. But at this moment let's remove the commented lines.